config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -105,7 +105,7 @@ postsubmits:
       decorate: true
       branches:
         - ^master$
-        - ^release-0.2$
+        - ^release-0.3$
```

Comment:
I'm wondering if we should just make these `^release-*` to avoid having to change everytime a new release branch is made?

Reply:
make sense, will update

Reply:
updated, thanks!